client/components/Navbar.js

```diff
@@ -1,26 +1,37 @@
-import React from 'react';
-import { connect } from 'react-redux';
-import { Link } from 'react-router-dom';
-import { logout } from '../store';
+import React from "react";
+import { connect } from "react-redux";
+import { Link } from "react-router-dom";
+import { logout } from "../store";
 
 const Navbar = ({ handleClick, isLoggedIn }) => (
-  <div>
-    <h1>Caffein-E Commerce</h1>
+  <div id="nav-bar">
```

Comment:
@Nievescs20 the nav-bar div now seems to overlay and hide the login & signup elements. Please update CSS positioning to those critical elements still display on the screen.

Reply:
this was a simple fix. taken care of